avoinspectorexample/src/main/java/app/avo/avoinspectorexample/MainActivity.java

```diff
@@ -119,9 +119,9 @@ public void intercept(Chain chain) {
 
 		avoInspector.hideVisualInspector(this);
 
-		avoInspector.showVisualInspector(this, DebuggerMode.bubble);
+		avoInspector.showVisualInspector(this, VisualInspectorMode.bubble);
 
-		DebuggerManager visualInspector = avoInspector.getVisualInspector();
+		DebuggerManager visualInspector = (DebuggerManager) avoInspector.getVisualInspector();
```

Comment:
This variable is never used. If this is here as an example on how to access it - then it should have a comment or something, otherwise the whole `exampleUsage` function makes no sense to me. 

The documentation should show all the functions and how to access them.
The code should show an actual real life example - not like this IMHO.